standards/out-of-hours.md

```diff
@@ -0,0 +1,60 @@
+
+These risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (i.e. without risking burnout), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
+
+Either kind of person, and all of those on the spectrum between, however, can easily burn out if a few core criteria are not met:
+
+- No one person is on call more than 50% of the time, and ideally no more than 30% of the time.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
+- The rota repeats regularly, so people know when they are on call over the coming weeks and months, and can plan accordingly.
+- People on call are appropriately paid for being available on call and for responding to incidents.
+- People on call know when they are expected to be available when on call, whether 24/7, 7am-10pm, or some other, agreed hours.
+- People on call can trade shifts with one another when they need to.
+- People on call for the first time are on call with someone experienced to being on call on their support rota, and ideally someone they know well.
+- People on call understand the circumstances under which they will be paged. This should ideally be a clearly defined list of alerts with agreed steps to resolve, but some services may not yet be in a position to support this.
```

Comment:
It probably won't be possible to provide agreed steps to resolve every situation. For example, a security breach could have unpredictable repercussions. Perhaps there is a clear and separate process to follow if a security breach is suspected and additional technical support would be available on call?

Reply:
Perhaps we need some basic guidance for teams on the different types of incident that may occur? All would require support from the team in or out of hours but the response could be quite different for each (application failure vs security breach for example)

Reply:
Yeah, that's why I say this:

> This should ideally be a clearly defined list of alerts with agreed steps to resolve, but some services may not yet be in a position to support this.

There probably should be (possibly already is?) guidance for dealing with generic classes of incident (like security breaches), but actually providing that guidance is outside the scope of this particular document.

Reply:
I think you're probably right re; scope.